src/components/card.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import styled from "styled-components"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import Category from "./category"
+
+import { Link } from "gatsby"
+
+export const ListNode = styled.div`
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    max-width: 22.4rem;
```

Comment:
Why do not use a file with css?

Reply:
We could add css modules, Guille suggested me the same. I did it this way as it is how it was being done in the rest of the project but it could be a good improvement to add. Do you think it's worth it to add it in this PR or maybe we can have another one refactoring that?

Reply:
@diegorosolino done!